File: StrykerDG.FarmForge.Client/farmforge_client/lib/models/dto/new_crop_log_dto.dart
class NewCropLogDTO {
  int cropId;
  int logTypeId;
  int cropStatusId;
  String notes;
  int unitTypeId;
  int quantity;

  NewCropLogDTO({
    this.cropId,
    this.logTypeId,
    this.cropStatusId,
    this.notes,
    this.unitTypeId,
    this.quantity
  });

  Map<String, dynamic> toMap() {
    return {
      'CropId': cropId,
      'LogTypeId': logTypeId,
      'CropStatusId': cropStatusId,
      'Notes': notes,
      'UnitTypeId': unitTypeId,
      'Quantity': quantity
    };
  }
}

File: StrykerDG.FarmForge.Api/Controllers/DTO/Requests/NewCropLogDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace StrykerDG.FarmForge.LocalApi.Controllers.DTO.Requests
{
    public class NewCropLogDTO
    {
        public int LogTypeId { get; set; }
        public int CropStatusId { get; set; }
        public string Notes { get; set; }
        public int? UnitTypeId { get; set; }
        public int? Quantity { get; set; }
    }
}
